stylesheets/_modal-cópia.scss

```diff
@@ -0,0 +1,90 @@
+     form {
```

Comment:
that file probably is an error. i do not have that in the style.scss being called. i do have _modal _and modalxs. in principle you can delete it